docs/tasks/issuers/setup-acme/dns01/cloudflare.rst

```diff
@@ -2,6 +2,10 @@
 Cloudflare
 =========================
 
+There are two ways to authenticate with Cloudflare. If you want to use
+API key you must have apiKeySecretRef defined, if you want to use API token
+you must have apiTokenSecretRef defined. If you provide both token will be used.
```

Comment:
Can you also make a note of the ambient credentials that can be used instead?
Looks like we can use
```
CLOUDFLARE_EMAIL
CLOUDFLARE_API_KEY
CLOUDFLARE_API_TOKEN
```